src/index.ts

```diff
@@ -48,9 +48,19 @@ export const ronin = (options: QueryHandlerOptions = {}) =>
     if (!c.env.RONIN_TOKEN)
       throw new Error("Missing `RONIN_TOKEN` in environment variables");
 
+    const userOptions = typeof options === "function" ? options() : options;
+    if (userOptions.token) {
+      console.warn(
+        "The `token` option is ignored in favour of `c.env.RONIN_TOKEN` when using the `ronin` middleware",
```

Comment:
```suggestion
        "The `token` option is ignored in favor of `c.env.RONIN_TOKEN` when using the `ronin` middleware.",
```

We default to US English! 🚀